server/dist/src/main/resources/concord-server.conf

```diff
@@ -418,6 +421,9 @@ concord-server {
 
         sshTimeoutRetryCount = 1
         sshTimeout = "10 minutes"
+
+        # default Git branch
+        defaultBranch = "master"
```

Comment:
why we need this default value if branch is mandatory? :)

Reply:
As either commitId or Branch is mandatory. Hence there is still possibility like branch may be null when `repositories` are created by giving only commitId. So i think, we would need this default value @ [1](https://github.com/walmartlabs/concord/blob/6d80f9d09bd10ffd578af5096211800c44111eda/server/impl/src/main/java/com/walmartlabs/concord/server/org/triggers/GithubTriggerEnricher.java#L88), [2](https://github.com/walmartlabs/concord/blob/6d80f9d09bd10ffd578af5096211800c44111eda/server/impl/src/main/java/com/walmartlabs/concord/server/process/pipelines/processors/RepositoryProcessor.java#L111) ... wdyt?